### ISSUE TITLE
mvn exec:exec isn't working

### ISSUE THREAD

**Issue by swarn2099** [alexa/alexa-avs-sample-app#269]
The command that's supposed to bring up the java interface doesnt work and innstead it gives me this error. I believe that my terminal directory is correct.

![image](https://cloud.githubusercontent.com/assets/16906029/19173310/509ea4fc-8bec-11e6-80a4-b126caba4a9f.png)

**Comment by jamesrequa** [comment]
I just posted the exact same issue :)

**Comment by tstephansen** [comment]
Add this to ~/.bashrc

`export M2_HOME=/opt/apache-maven-3.3.9`
`export M2=$M2_HOME/bin`
`export PATH=$M2:$PATH`

Save .bashrc and run source ~/.bashrc from the terminal (or close all terminals and reopen)

**Comment by swarn2099** [comment]
Can you elaborate?

**Comment by swarn2099** [comment]
Can you elaborate? @tstephansen

**Comment by tstephansen** [comment]
Open terminal and type "cd ~" and press enter. Then type "nano .bashrc" and press enter. Scroll down to the bottom of the file and add the following lines:

`export M2_HOME=/opt/apache-maven-3.3.9`
`export M2=$M2_HOME/bin`
`export PATH=$M2:$PATH`

Press CTRL+X then type y and hit enter.

At this point you can either quit all instances of terminal or type "source ~/.bashrc" to set the environment variables. 

Go back to the samples/javaclient/ directory and run mvn exec:exec and it should run. If it doesn't run then type the following commands.

mvn validate
mvn install
mvn exec:exec

Then you should be good.

**Comment by ajotwani** [comment]
We're closing this Issue since the original poster (or another commenter) hasn't yet responded to the question or request made to them 14 days ago. We therefore assume that the user has lost interest or resolved the problem on their own.

Don't worry though; if this is an error, let us know with a comment and we'll be happy to reopen the Issue.

Thanks!

**Comment by keleric** [comment]
Someone else had this issue. Maybe this will help future people with this problem. This is how he resolved it:

"Previously I installed maven manually so there was a maven.sh script in my /etc/profile.d which was setting that environment variable.  I uninstalled maven and deleted that script then re-installed maven and it's good to go now.

sudo apt-get remove maven
rm /etc/profile.d/maven.sh
. automated_install.sh"
